packages/core/src/index.ts

```diff
@@ -428,6 +432,12 @@ class Hopr extends EventEmitter {
     }
 
     const currentChannels = await this.getAllChannels()
+    verbose('Channels obtained', currentChannels)
+
+    if (!currentChannels) {
```

Comment:
is it an empty array check or null check?

Reply:
`null` check, both `![]` and `![...]` return `false`.

Reply:
I would say for the sake of clarity make it verbose if you *know* the type it shouldn't be a prob, right?

Reply:
Updated in 5ad47256100797d3c37b1726d0a429d9b37ae695

Reply:
FWIW we should have this behind a [Maybe like](https://engineering.dollarshaveclub.com/typescript-maybe-type-and-module-627506ecc5c8) data structure.